src/main/java/bio/terra/cloudevents/GCSEvent.java

```diff
@@ -0,0 +1,324 @@
+package bio.terra.cloudevents;
+
+import java.util.Date;
+
+public class GCSEvent {
```

Comment:
Do you need to write this class yourself? Can you use `com.google.events.cloud.pubsub.v1.Message` or `com.google.events.cloud.storage.v1. StorageObjectData` or one of the other events from 
https://github.com/googleapis/google-cloudevents-java instead?

Reply:
Created story [QA-1522](https://broadworkbench.atlassian.net/browse/QA-1522) to work on this feedback.